### DISCUSSION TITLE
PyMuPDF-1.24.14 released

### DISCUSSION THREAD

**Discussion by julian-smith-artifex-com** [pymupdf/PyMuPDF#4063] [Announcements]
PyMuPDF-1.24.14 has been released.

Wheels for Windows, Linux and MacOS, and the sdist, are available on pypi.org and can be installed in the usual way, for example:

```
python -m pip install --upgrade pymupdf
```

[Linux-aarch64 wheels will be built and uploaded later.]

**Changes in version 1.24.14 (2024-11-19)**

* Use MuPDF-1.24.11.

* Fixed issues:

  * **Fixed** #3448
  * **Fixed** #3758
  * **Fixed** #3813
  * **Fixed** #3933
  * **Fixed** #4018
  * **Fixed** #4043
  * **Fixed** #4047
  * **Fixed** #4050

* Other:

  * Ensure that words from `Page.get_text()` never contain RTL/LTR char mixtures.
  * Fix building with system MuPDF.
  * Add dot product for points and vectors.

<hr /><em>This discussion was created from the release <a href='https://github.com/pymupdf/PyMuPDF/releases/tag/1.24.14'>PyMuPDF-1.24.14 released</a>.</em>

**Comment by julian-smith-artifex-com** [comment]
Linux-aarch64 wheels are now available; install in the usual way with pip.

**Comment by kkk935208447** [comment]
Thanks for the update
